tools/wake-format/emitter.cpp

```diff
@@ -1591,15 +1591,13 @@ wcl::doc Emitter::walk_flag_global(ctx_t ctx, CSTElement node) {
 
 wcl::doc Emitter::walk_guard(ctx_t ctx, CSTElement node) {
   MEMO(ctx, node);
-  MEMO_RET(walk_placeholder(ctx, node));
```

Comment:
I assume `walk_placeholder` is used elsewhere and so doesn't become dead code?

Reply:
yeah, its the main fallback function